FSInteractiveMap/FSInteractiveMap/FSInteractiveMapView.m

```diff
@@ -45,6 +45,47 @@ - (void)setDefaultParameters
 
 #pragma mark - SVG map loading
 
+- (void)loadMap:(NSString*)mapName forSize:(CGSize)size withColors:(NSDictionary*)colorsDict {
```

Comment:
There is a log of duplication in that method from - (void)loadMap:(NSString*)mapName withColors:(NSDictionary*)colorsDict. Can you try to factor it?